CHANGELOG.md

```diff
@@ -6,9 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [[Unreleased]]
+
+## [1.1.2] - 2021-11-02
+### Added
+- Support for Python 3.10
```

Comment:
this should be 1.2.0 because we're adding something

Reply:
It still feels like a patch because no code was updated